pkgs/top-level/python-packages.nix

```diff
@@ -2993,6 +2993,23 @@ in {
 
   };
 
+  hiro = buildPythonPackage rec {
```

Comment:
We are moving Python expressions out of `pkgs/top-level/python-packages.nix` into `pkgs/development/python-modules/<module>/default.nix`. 

Please move the expression there, and call it from `pkgs/top-level/python-packages.nix` using `callPackage ../development/python-modules/<package> { };`.

Reply:
Aha - I'd misunderstood as applying to the keyrings set until you'd realized that alt was only a child to get the `.`. I'll take this on presently.